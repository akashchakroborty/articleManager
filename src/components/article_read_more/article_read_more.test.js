import React from "react";
import { shallow } from "enzyme";
import ArticleReadMore from "./article_read_more.component";

describe("ArticleReadMore component", () => {
  let wrapper;
  beforeEach(() => {
    const mockProps = {
      article: {
        title: "Mock data",
        description: "Mock data",
        author: "Mock data",
        content: "Mock data",
        id: "Mock data",
        img: "Mock data",
        avatar: "Mock data",
        claps: 38,
      },
    };

    wrapper = shallow(<ArticleReadMore {...mockProps} />);
  });

  it("should render ArticleReadMore component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
