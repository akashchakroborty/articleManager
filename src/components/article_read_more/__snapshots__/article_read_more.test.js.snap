// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleReadMore component should render ArticleReadMore component 1`] = `ShallowWrapper {}`;
